test/Tests.Infrastructure/TestBase.cs

```diff
@@ -471,6 +470,18 @@ public bool DeletePrevious
                 }
             }
 
+            private bool _ignoreProcessorAffinityChanges = true;
+
+            public bool IgnoreProcessorAffinityChanges
```

Comment:
Should not be at any point configurable. We cannot run tests that affect processor affinity.

Reply:
done